docs/woocommerce.com/activity-panels.md

```diff
@@ -0,0 +1,79 @@
+# Activity Panels
+
+The Activity Panel aims to be a "one stop shop" for managing your store - fulfill new orders, manage product inventory, moderate reviews, and get information about running your store.
+
+Activity Panels can be accessed wherever the WooCommerce Admin nagivation bar is shown.
+
+![Activity Panels Tabs Overview](images/activity-panels-tabs.png)
+
+There are five panels: Inbox, Orders, Stock, Reviews, and WordPress Notices. You can read more about each panel below.
+
+Each panel has an orange indicator denoting "unread" items.
+
+![Activity Panels Button on Mobile](images/activity-panels-mobile-view.png)
+
+On mobile devices, all of the panels get collapsed under the Woo logo.
+
+## Inbox Panel
+
+![Inbox Activity Panel](images/activity-panels-inbox.png)
+
+This Inbox Panel is dedicated to informational content coming from multiple sources such as WooCommerce core, WooCommerce.com Subscription management, extensions activity and store achievements. It also displays insightful content that could help with the day to day tasks of managing and optimizing a store.
+
+## Orders Panel
+
+![Orders Activity Panel](images/activity-panels-orders.png)
+
+The Orders Panel displays Orders that are in need of fulfillment. You can click "begin fulfilment" to go to the Order Details screen.
```

Comment:
This should have more info 
- contains orders that are actionable. 
- default is needs fulfillment.

Reply:
Done in 28ee4ab0.